nuxeo-services/nuxeo-platform-convert/src/test/java/org/nuxeo/ecm/platform/convert/tests/TestSOfficeConverter.java

```diff
@@ -46,18 +52,40 @@ public void testConverter() throws Exception {
         checkConverterAvailability(converterName);
         checkCommandAvailability("soffice");
 
+        BlobHolder result = convertTextBlob(converterName);
+        checkPDFDocument(result);
+        checkPDFDocument(result);
```

Comment:
Why call it twice? If this is on purpose please add a comment explaining why.

Reply:
My bad, not needed :)